diddiparser2/parser.py

```diff
@@ -59,12 +59,15 @@ def print_command(self, cmd):
 
     def executeline(self, line):
         "Parse, read and run a single line of code."
-        line = line.replace('("', "(")
-        line = line.replace("('", "(")
-        line = line.replace('")', ")")
-        line = line.replace("')", ")")
         parsed_line = line.replace(");", "")
-        call, arg = parsed_line.split("(")[0], parsed_line.split("(")[1]
+        call = parsed_line.split("(")[0]
+        pos = len(f"{call}(")  # use this to avoid conflicts
+        arg = parsed_line[pos:]
+        del pos  # delete the aux

```

Comment:
The `pos` variable is just an auxiliar variable, to avoid the conflict between linters.